sentry-samples/sentry-samples-spring/build.gradle.kts

```diff
@@ -0,0 +1,40 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    id(Config.BuildPlugins.springBoot) version Config.springBootVersion
+    id(Config.BuildPlugins.springDependencyManagement) version Config.BuildPlugins.springDependencyManagementVersion
+    kotlin("jvm")
+    kotlin("plugin.spring") version Config.kotlinVersion
+}
+
+group = "io.sentry.sample.spring"
+version = "0.0.1-SNAPSHOT"
+java.sourceCompatibility = JavaVersion.VERSION_1_8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
```

Comment:
can we reuse the things that already live in the `Config` class?

Reply:
I guess I already said that, these samples are to easier development, not for customers.

see an example: https://github.com/getsentry/sentry-android/blob/master/sentry-android-timber/build.gradle.kts#L74

there's no need to hardcode when they already live in the Config class.
If we decide using only hardcoded strings, I'm fine with that too, but rather do it everywhere and kill the Config class for good.

those generated samples and closer to real projects will live here https://github.com/getsentry/examples/tree/master/java

Reply:
We are already reusing things from `Config` class to set the dependency management and plugins. There are no dependencies currently set in config class that we could directly replace here.

Reply:
```
    implementation("org.springframework.boot:spring-boot-starter-security")
    implementation("org.springframework.boot:spring-boot-starter-web")
    implementation("org.springframework.boot:spring-boot-starter")
    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
```

I see those there

https://github.com/getsentry/sentry-android/blob/feat/sentry-java/buildSrc/src/main/java/Config.kt

Reply:
In `Config` they are listed under `TestLibs` as they are used to test integrations. Should I refer them from `TestLibs` in samples or move them to `Libs`?

Reply:
I'd move to `Libs`, guess it makes more sense, we have `Timber` there too

Reply:
Done.